dummy-project/src/features.py

```
@@ -2,3 +2,25 @@ def add_feature_length(df, column_name):
     """Adds a column with the length of text in the specified column."""
     df[f'{column_name}_length'] = df[column_name].astype(str).apply(len)
     return df
+
+def normalize_column(df: pd.DataFrame, column: str) -> pd.DataFrame:
```

Ruff (0.8.2)

Undefined name `pd`

(F821)